src/SourceCode.Chasm.Repository.Disk/DiskChasmRepo.cs

```diff
@@ -42,7 +51,7 @@ public DiskChasmRepo(string rootFolder, IChasmSerializer serializer, Compression
             // Refs
             {
                 const string container = "refs";
-                var path = Path.Combine(rootPath, container) + Path.DirectorySeparatorChar;
```

Comment:
Appending a '/' here means we don't need to check for it every time we create a file

Reply:
It's not platform agnostic. `Path.Combine` is the only way paths should be combined.